lib/system.nim

```diff
@@ -781,8 +781,16 @@ proc excl*[T](x: var set[T], y: T) {.magic: "Excl", noSideEffect.}
   ##  b.excl(5)  #=> {2,3,6,12,545}
 
 template excl*[T](s: var set[T], flags: set[T]) =
-  ## excludes the set of flags to ``x``.
+  ## excludes the set of flags to ``s``.
   s = s - flags
+  
+template `+=` *[T](s: var set[T], y: set[T]|T) =
+  ## Alias for ``incl``.
+  s.incl(flags)
+
+template `-=` *[T](s: var set[T], y: set[T]|T) =
+  ## Alias for ``excl``.
+  s.excl(flags)
 
```

Comment:
I suggest you to add tests, too. In both procs you have parameter 'y', but use 'flags' in the body

Reply:
whoops lol. I broke my local git repo after I had tested it so I just retyped it in the online github editor...should've double-checked